### PR TITLE
Announce Camel Quarkus 2.13.0 release

### PR DESCRIPTION
**Pull request by zbendhiba**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: content/releases/q/release-2.13.0.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+url: "/releases/q-2.13.0/"
+date: 2022-09-26
+eol: 2022-03-26
```
Comment (aldettinger) [inline content/releases/q/release-2.13.0.md:4]: Is it like end of life in the past ?

Comment (zbendhiba) [reply content/releases/q/release-2.13.0.md:4]: yes I checked what we did in 2.7.x, we did set for 6 months

**Comment by zbendhiba** [comment]
For the error we did change the tag id for a section that is refferred inside Camel-k docs. The thing that is weird is that it's also broken for oldest Camel K documentations, and those shouldn't point the Camel quarkus 2.13.x.

File: content/releases/q/release-2.13.0.md (hunk 1 above)
Comment (zbendhiba) [reply content/releases/q/release-2.13.0.md:4]: https://github.com/apache/camel-website/blob/main/content/releases/q/release-2.7.0.md

Comment (aldettinger) [reply content/releases/q/release-2.13.0.md:4]: date-fomats wise, it looks that in [2.7.x](https://github.com/apache/camel-website/blob/main/content/releases/q/release-2.7.0.md), eol > date.
So would we need to put `eol: 2023-03-26` ?

Comment (zbendhiba) [reply content/releases/q/release-2.13.0.md:4]: oh okay. Nice catch

**Review by davsclaus — APPROVED** [review]

**Review by aldettinger — APPROVED** [review]
That's great, just a few CI issues to be fixed and we are live :+1:

**Comment by zbendhiba** [comment]
Opened a PR to fix doc in Camel-k, but I'm not sure if this is the right fix : https://github.com/apache/camel-k/pull/3698

**Comment by zregvart** [comment]
Seems that the older versions of the Camel-K documentation point to the latest version Camel-Quarkus. For example [in 1.6.x](https://github.com/apache/camel-k/blob/release-1.6.x/docs/modules/ROOT/pages/observability/monitoring/integration.adoc?plain=1#L41). We usually introduce the version in those links via [asciidoc attribute](https://github.com/apache/camel-k/blob/release-1.6.x/docs/antora.yml#L36). So for branches 1.6.x...1.10.x the anchor id can remain the same if the xref is changed to something like `xref:{camel-quarkus-version}@camel-quarkus::reference/extensions/microprofile-metrics.adoc[Camel Quarkus MicroProfile Metrics extension]`

**Pull request merged**